packages/zoe/src/zoeService/utils.ts

```diff
@@ -1,17 +1,16 @@
+// Ambient types
```

Comment:
These aren't types imports so this module will actually import those JS files. They happen to be empty but that doesn't matter.

The reason this is okay is because there are no runtime imports of this module.